include/seqan3/io/alignment_file/detail.hpp

```diff
@@ -87,21 +87,21 @@ template <typename reference_char_type, typename query_char_type>
     requires std::detail::weakly_equality_comparable_with<reference_char_type, gap> &&
              std::detail::weakly_equality_comparable_with<query_char_type, gap>
 //!\endcond
-char compare_aligned_values(reference_char_type const reference_char,
-                            query_char_type const query_char,
-                            bool const extended_cigar)
+cigar_op compare_aligned_values(reference_char_type const reference_char,
+                                query_char_type const query_char,
+                                bool const extended_cigar)
 {
     return (reference_char == gap{})
                 ? (query_char == gap{})
-                    ? 'P'
-                    : 'I'
+                    ? 'P'_cigar_op
```

Comment:
This will probably be faster depending on the compiler optimization for ternary expressions but I do not find this more readable. Especially when you are not accustomed to bit operations you really have to think about what's happening here whereas in if-else (ternary) expressions you just read it.

I would say we wait for the benchmark to see if this makes a performance difference but since I do not want to stall this PR I'll just include your changes